src/sparseml/exporters/transforms/kv_cache/configs.py

```diff
@@ -125,6 +125,16 @@ class Config:
     multiply_batch_by_num_att_heads=False,
 )
 
+GPT_NEO_CONFIG = KeyValueCacheConfig(
+    model_name="gpt_neo",
+    additional_transforms=AdditionalTransformsCodeGen,
```

Comment:
hmm...a bit afraid that in the future, potentially, due to a refactor or model changes, we might change the codegen transforms.  Because there is no explanation in the code as to why we decided to reuse those transforms for the gpt neo config, once something breaks, it will be difficult to understand our decision. I propose adding a line of code like

```python
# reusing the transforms for codegen, because it happens to match what we need for gpt neo
additional_transforms_gpt_neo = AdditionalTransformsCodeGen
...
KeyValueCacheConfig(additional_transforms=additional_transforms_gpt_neo, ...)
```